hack/update-deps.sh

```diff
@@ -55,6 +55,8 @@ drop-dep vendor/golang.org/x/text/language vendor/golang.org/x/text/internal
 drop-dep vendor/google.golang.org/api/transport/grpc vendor/google.golang.org/api/transport
 drop-dep vendor/github.com/golang/protobuf/protoc-gen-go/grpc vendor/github.com/golang/protobuf/protoc-gen-go
 drop-dep vendor/github.com/golang/protobuf/protoc-gen-go/generator vendor/github.com/golang/protobuf/protoc-gen-go
+drop-dep vendor/github.com/golang/protobuf/protoc-gen-go/generator vendor/github.com/golang/protobuf/protoc-gen-go
```

Comment:
This line is a dupe of the one above it?

Reply:
Done

Reply:
did you forget to push? I still see the duplicate here.

Reply:
Done. So confused...